manifests/profile/ruby.pp

```diff
@@ -85,4 +85,20 @@
       }
     }
   }
+
+  $::ruby_versions.each |$version| {
+    unless $version in $supported_versions {
+      unless $version == $global_version {
+        exec { "rbenv uninstall ${version}":
+          command     => "rbenv init -; rbenv uninstall ${version}",
```

Comment:
Does this need to be `eval "$(rbenv init -)"` or equivalent?

Reply:
@daaang ^?

Reply:
It was chicken-waving, so I just got rid of it.